lib/_utils/constants/string_constants.dart

```diff
@@ -38,5 +38,12 @@ class AppConst {
   static const fastEasyWayToPrepare = "Fast & easy way to prepare.";
   static const selectExamCategory = "Select exam category";
   static const submit = "Submit";
-  static const collectionName = "exam_list";
```

Comment:
Collection or db related any string must be in different class "DbConstants" in same file. @Astha186 @anamika2812 @AvnishSharma-max @iramkshukla